benefits/core/session.py

```diff
@@ -152,14 +152,22 @@ def logout(request):
     update(request, logged_in=False, enrollment_token=False)
 
 
+def oauth_claims(request):
+    """Get the oauth claims from the request's session, or None"""
+
+    return [claim for claim, value in OAuthSession(request).claims_result.verified.items() if value] or None
+
+
```

Comment:
We did not want to reintroduce this function, it is not needed outside of the usage in `oauth_extra_claims`. We are trying to simplify usage of `session`

* #2800
* https://github.com/Office-of-Digital-Services/django-cdt-identity/issues/50

Reply:
To be clear: keep the function deleted as it was already, simplify `session.oauth_extra_claims` as we talked about in the other thread. We'll come back to those session tickets later.

Reply:
You can see we had a long PR discussion about this previously: https://github.com/cal-itp/benefits/pull/2783#discussion_r2014560634

Reply:
Ah, that makes sense. I removed `session.oauth_claims` and just inlined its behavior into `session.oauth_extra_claims`. The rest of `session.oauth_extra_claims` was simplified so I think this is probably ready.